modules/rosa-view-cloudwatch-logs.adoc

```diff
@@ -0,0 +1,22 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// rosa_logging/rosa-install-logging.adoc
+
+[id="rosa-view-cloudwatch-logs_{context}"]
+= Viewing forwarded logs
+
+.Prerequisites
+
+* The `cluster-logging-operator` add-on service is installed.
+* The Amazon CloudWatch add-on service is installed.
+
+.Procedure
+
+To view logs that have been forwarded to Amazon Web Services (AWS) from {product-title}:
+
+. Log in to the AWS console.
+. Select the region the cluster is deployed in.
+. Select the `CloudWatch` service.
```

Comment:
These 3 items in these two steps that are from a GUI should be in bold, not backticks. So `*CloudWatch*`, `*Logs*`, and `*Log Groups*`.

Reply:
Fixed.